### DISCUSSION TITLE
Possible line duplication?

### DISCUSSION THREAD

**Discussion by megtog** [ConsenSys-archive/ganache#3789] [General]
Is the line `this.blockIndexes.close()` duplicated?

https://github.com/trufflesuite/ganache/blob/ec09d2715a94e15ab58a48eb64e89bea86efd569/src/chains/ethereum/ethereum/src/database.ts#L165-L176

**Comment by davidmurdoch** [comment]
Looks like that second line is supposed to be `this.blockLogs.close()`. Thanks for catching this!

**Reply by davidmurdoch** [reply]
I've opened an issue for this: https://github.com/trufflesuite/ganache/issues/3792
